components/com_joomgallery/controllers/comments.php

```diff
@@ -54,6 +54,15 @@ public function comment()
    */
   public function remove()
   {
+    // Check whether we are allowed to delete
+    $canDo = JoomHelper::getActions();
```

Comment:
> Why I haven't seen that?

Maybe you did just copy the functioning peace of code in the backend and pasted it just in the frontend without testing it 😉 ?

I know that writing code is much more fun than testing code. But in the real world testing code is so important because not working code may cost o lot of money (see WhatsApp last week).

Reply:
Normally I try to test the piece of code I changed. But I guess my development environment is not set up optimally. After every change I have to copy the file with the chnaged code from the repository folder to the corresponding folder on the WAMP server...

Reply:
> After every change I have to copy the file with the chnaged code from the repository folder to the corresponding folder on the WAMP server...

That sounds very complex, but also prone to errors.